bitnami/dataplatform-bp2/values.yaml

```diff
@@ -92,7 +92,7 @@ kafka:
     ##
     jmx:
       ## @param kafka.metrics.jmx.enabled Enable JMX exporter for Kafka
-      enabled: true
+      enabled: false
```

Comment:
These changes require a major bump of the version, since an upgrade will change the default values of your release

Reply:
Since this chart was released yesterday, we are going to proceed without bumping the major.